packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -757,7 +758,7 @@ export function TransferPanelMain({
   }, [actions.app, isDepositMode, selectedToken, token])
 
   type NetworkListboxesProps = {
-    from: Omit<NetworkListboxProps, 'label'>
+    from: Omit<NetworkListboxProps, 'label' | 'options'>
```

Comment:
why do we omit options?

Reply:
actually we only need onChange now, i'll remove the additional code